Azure/brigade-charts
# fix(charts): fix brigade-0.20.0 artifact

This fixes the Brigade 0.20.0 chart.

I'd inadvertently included working changes (untracked sub-charts in `charts/brigade/charts` from https://github.com/Azure/brigade-charts/pull/22) into the PR updating this repo's index.yaml (https://github.com/Azure/brigade-charts/pull/24) causing some charts to be bundled in prior to completion of the refactor PR.

To test:

Check out this branch and install a fresh instance of Brigade via its corresponding chart artifact:

```console
$ helm install -n brigade docs/brigade-0.20.0.tgz
```

Confirm output as below (shows default/classic `-github-gw` service, etc.):

```console
==> v1/Service
NAME                       TYPE          CLUSTER-IP      EXTERNAL-IP  PORT(S)         AGE
brigade-kashti             ClusterIP     10.104.81.226   <none>       80/TCP          1s
brigade-brigade-api        ClusterIP     10.97.51.211    <none>       7745/TCP        1s
brigade-brigade-github-gw  LoadBalancer  10.101.156.183  <pending>    7744:31638/TCP  1s
```
